pkg/server/server.go

```diff
@@ -362,6 +364,21 @@ func (s *Server) ListObjects(ctx context.Context, req *openfgav1.ListObjectsRequ
 			User:                 req.User,
 		},
 	)
+	if err == nil {
```

Comment:
Can we stay within standard Go idioms here and favor the `non-nil` check instead? That's what we mostly do elsewhere, and changing that pattern here makes it non-standard a tad.

```go
if err != nil {
    return nil, err
}

// otherwise do the other stuff and
return &openfgav1.ListObjectsResponse{...}, nil
```